content/html/concepts/tags/terms/object/object.md

```diff
@@ -0,0 +1,44 @@
+---
+Title: '<object>'
+Description: 'Represents some resource external to the HTML document that can be treated as an image, a nested browsing context, or content to be handled by a browser plugin.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Tags'
+  - 'Parameters'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `<object>` tag represents some resource external to the HTML document. This resource can be treated as an image, an embedded browsing context, or as content to be handled by a browser plugin.
+
+The `<object>` tag requires at least the `type` and `data` attributes specified. The `type` attribute tells the browser what MIME type of content to be displayed, such as "application/pdf", "application/x-java-applet" or "audio/mp4". The `data` attribute contains the URL to the resource being used.
+
+An `<object>` element may contain zero or more [`<param>`](https://www.codecademy.com/resources/docs/html/tags/param) elements to pass parameters on to the resource it is displaying.
+
+## Syntax
+
+```html
+<object type="MIME type" data="URL of resource"></object>
+```
+
+With a parameter:
+
+```html
+<object type="MIME type" data="URL of resource">
+  <param name="parameter name" value="parameter value" />
+</object>
+```
+
+## Example
+
```

Comment:
I wonder if directly embedding that `<obejct>` tag below this snippet would be able to broadcast that YouTube video from within the entry?

It would be cool to get some more interactivity into these entries. Maybe if that doesn't work, what about an `<iframe>` tag?

```
<iframe
  src="https://www.youtube.com/watch?v=138enunVlfs"
  width="420"
  height="315"
>
  Alt-text for Codecademy video about HTML
</iframe>
```

Reply:
Oh well, worth a try 😄